torchtune/modules/peft/lora.py

```diff
@@ -123,6 +119,8 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         """
         if self._quantize_base:
             out = linear_nf4(input=x, weight=self.weight)
+            if self.use_bias:
```

Comment:
Why do we use torchao linear_nf4 here and not our own?

Reply:
Not sure I follow.. we don't have our own linear_nf4. Even our `FrozenNF4Linear` is just a wrapper around their `linear_nf4`

Reply:
I meant, why do we not reuse the FrozenNF4Linear class here so we don't have to define this bias solution in multiple places?

Reply:
Discussed a bit offline. It's a nice idea but going to punt on it for now